docs/user_guide/datasets.rst

```diff
@@ -0,0 +1,332 @@
+    ... 'precision': skm.precision_score,
+    ... 'recall': skm.recall_score,
+    ... 'false positive rate': false_positive_rate,
+    ... 'true positive rate': true_positive_rate,
+    ... 'selection rate': selection_rate, 
+    ... 'count': count}
+    >>> grouped_metric = MetricFrame(metrics=metrics,
+    ... y_true=y_test, 
+    ... y_pred=predicted,
+    ... sensitive_features=X_test["B"])
+    >>> print(grouped_metric.by_group)
+        accuracy precision    recall false positive rate true positive rate  \
+    B                                                                            
+    False  0.852459      0.92  0.766667            0.064516           0.766667   
+    True   0.863636  0.909091  0.833333                 0.1           0.833333   
+
+        selection rate count  
+    B                           
+    False       0.409836    61  
+    True             0.5    66  
+
+    >>> grouped_metric = MetricFrame(metrics=metrics,
+    ... y_true=y_test,
+    ... y_pred=predicted,
+    ... sensitive_features=X_test["LSTAT"])
+    >>> print(grouped_metric.by_group)
+        accuracy precision    recall false positive rate true positive rate  \
+    LSTAT                                                                        
+    False  0.901639  0.910714  0.980769            0.555556           0.980769   
+    True   0.818182       1.0  0.142857                 0.0           0.142857   
+
+        selection rate count  
+    LSTAT                       
+    False       0.918033    61  
+    True        0.030303    66  
+
+
+.. _discussion:
+
+Discussion
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Boston housing dataset is rife with ethical issues, and in general, we 
+strongly discourage using it in predictive modelling analyses. We've kept it 
+in fairlearn because of its potential as a teaching tool for how to deal with 
+ethical issues in a dataset. There are ways to `remove correlations between 
+sensitive features and the remaining columns 
+<https://scikit-lego.netlify.app/fairness.html>`_, but other benchmark datasets
+exist that do not present these issues.
+
+It's important to keep the differences between the way Harrison and Rubenfield 
+used the dataset and the way modern machine learning practicioners have used 
+it in focus. Harrison and Rubenfield conducted an empirical econometric study,
+the goal of which was to determine the causal impacts of these variables on 
+median home value. Interpretation of causal models involves looking at model
+coefficients to ascertain the effect of one variable on the depedent variable,
+holding all other factors constant. This use case is different than the typical 
+supervised learning analysis. A machine learning model will pick up on the 
+patterns encoded in the data and use that to predict an outcome.
+In the Boston housing dataset, the patterns the authors encoded through
+the *B* and *LSTAT* variables include systemic racism and class inequalities, 
```

Comment:
Yes, definitely!

Reply:
Hi @riedgar-ms, here is what I added: "*LSAT* and *B* are not mutually exclusive. One might infer that "lower-status" Black people impact house prices differently than non-"lower-status" Black people. Additionally, Black people might be more likely to be categorized  as "lower status" by the author's definition due to systemic racism.  Harrison and Rubenfield do not consider this intersectionality in their analysis; this phenomenon could be measured via an interaction variable between the two fields.  However, including only one of these variables in the analysis is not sufficient in removing the bias encoded in the removed variable from the dataset. Because these columns are related, one likely can serve as a proxy for the other. Thus, we recommend great care be taken to account for intersectionality in data."

Please let me know if you suggest different phrasing!

Reply:
I'm not a fan of the first couple of sentences, but I like the rest of it. How about starting with something like:

> Intersectionality also requires consideration. Due to systematic racism, black people may be more likely to be categorized as "lower status" by the authors' definition. Harrison and Rubenfield do not consider....

At the very least, I don't think that 'mutually exclusive' (I'd personally prefer 'orthogonal' but others might disagree) and 'might infer' are great word choices. I'm not sure what I'd suggest as an alternative for the second, though.

And a really nitpicking point.... if there's more than one author, I believe the possessive has the apostrophe _after_ the 's.'

Reply:
Edited paragraph:

"Intersectionality also requires consideration. Due to systematic racism, Black people may be more likely to be categorized as "lower status" by the authors' definition. Harrison and Rubenfield do not consider this intersectionality in their analysis. In an econometric analysis, intersectionality could be measured via an interaction variable between the two fields. Including only one of these variables in the analysis is not sufficient in removing the bias encoded in the removed variable from the dataset. Because these columns are related, one likely can serve as a proxy for the other. Thus, we recommend great care be taken to account for intersectionality in data."

Reply:
I like it (obviously).... @hildeweerts and @romanlutz ?